deps/build.jl

```diff
@@ -1,47 +1,37 @@
-if Sys.KERNEL === :Darwin && (!success(`command -v xcode-select`) || isempty(readchomp(`xcode-select -p`)))
-    error("Building Rmath on macOS requires the Xcode command line tools to be installed.\n",
-          "You can install them from the command line using `xcode-select --install`.")
+using BinaryProvider
+
+# This is where all binaries will get installed
+const prefix = Prefix(!isempty(ARGS) ? ARGS[1] : joinpath(@__DIR__,"usr"))
+
+libRmath = LibraryProduct(prefix, String["libRmath-julia"])
+products = [
+libRmath
+]
+
+# Download binaries from hosted location
+bin_prefix = "https://github.com/JuliaStats/Rmath-julia/releases/download/v0.2.2"
+
+# Listing of files generated by BinaryBuilder:
+download_info = Dict(
+    BinaryProvider.Linux(:aarch64, :glibc) => ("$bin_prefix/rmath.aarch64-linux-gnu.tar.gz", "0e56d5c5472ef832f76949e863958676c272eeaa2405b3239930e1e0b8d6313d"),
```

Comment:
My remark it completely unrelated to this PR, but I wonder why one needs to list manually files for all architectures. Wouldn't it be possible for BinaryBuilder to automatically generate a file with this list (and checksums), and download that file to get this information? Then only one URL/checksum pair would have to be listed here, and platform details wouldn't pollute the code.

Reply:
The file does get generated and uploaded (look at the release artifacts here: https://github.com/JuliaStats/Rmath-julia/releases). We could certainly just download and execute that, though I do like this because it's very explicit. I think it's fine to go with this style for now and maybe revisit it once we have some more experience.

Reply:
OK, great. I wasn't too concerned about this package, but more about less experienced package authors who might be scared by all the platform-specific code.